conferences/abstracts/2021-01-RightsCon.md

```diff
@@ -0,0 +1,89 @@
+RightsCon2021 uses a platform for submissions - these are the questions!
+
+## About You
+Primary contact: Laura Carter
+Secondary contact: Malvika Sharan
+
+## About Your Session
+
+### Which program category does your session fall under? (select from drop-down menu)
+
+**Human Rights-centred Design**: 
+This category fosters discussion on technology design, tool development, and user education and experience. 
+It also covers openness and accessibility, community-driven innovation, co-design processes, and user research that supports and prioritizes human rights principles. 
+
+### Does your session touch on one of our intersecting themes? 
+_(choose one from: Public health,
+Protest,
+Indigenous rights,
+Environmental sustainability,
+Racial justice,
+Gender justice,
+Youth and children's rights,
+Migration)_
+
+- No
+
+### Session title (in English)
+_20 words_ 
+_Pique the audience's interest while also explainining the 'big idea' of your session_
+
+**The Turing Way: Building a collaborative guide to ethical data science**
+
+### Which institution(s) will host your session?
+_Write the full name of the institution(s) that should be credited publicly as the host of the session._
+_If you have more than one host, separate them using a semicolon (e.g. "Access Now; RightsCon"). If your session is not affiliated with an institution, you can write “Independent.”_
+
+**The Alan Turing Insitute**
+
+## Format and Goal
+
+### Session format (choose from drop-down)
+
+**Tech Demo** - A brief (8-10 minute), prepared presentation on a new technology, tool, or technical project. Demos are recorded in advance of the summit.
+
+## Session Concept 
+
+### What’s your product, initiative, or innovation?  (150 words)
+_In a few sentences, tell us what you plan to demo. (Yes, we know it’s not much space – make it count!)_
+
+The Turing Way is a community, and a global collaboration.
+This presentation will demonstrate one of the five open-source, online, collaborative books that form the project: the Guide to Ethical Research.
+The other four guides cover Reproducible Research, Project Design, Communication, and Collaboration.
+We will talk about the aims of the project, the community we are building, and, most importantly: how you can get involved!
```

Comment:
```suggestion
In data science, we make data-driven decisions throughout the lifecycle of their project . 
It is our duty to ensure that our work maintains a high level of moral integrity as well as the highest scientific standards, especially when it impacts people’s lives.
In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
In particular, we will demonstrate its guide for ethical research, the community behind its development, and, most importantly, how RightsCon attendees can get involved. (101/150 words)
```

Reply:
We have a lot of space to explain the ethical guide - as in, would you like to mention the titles of the chapters you have planned?